src/components/Scrollable/Scrollable.tsx

```diff
@@ -251,4 +251,4 @@ function prefersReducedMotion() {
   }
 }
 
-export default withAppProvider({withinScrollable: true})(Scrollable);
+export default withAppProvider<Props>({withinScrollable: true})(Scrollable);
```

Comment:
There's an issue somewhere (in the old repo) with `Scrollable` being typed as any. Previous HoC changes + this should add it's types back. Without this were now running into type errors.